### PR TITLE
Use table to list programs and services

### PR DESCRIPTION
**Pull request by rubenwardy**
- [x] Use same table theme as serverlist, currently I'm using http://codepen.io/alassetter/pen/cyrfB.html
- [x] document B3DExport
- [x] document wiki and dev wiki

screenshot_2017-03-11_21-07-11

### CODE REVIEW

File: development.html [hunk 1]
```diff
@@ -70,53 +70,83 @@ <h1 id="reporting-issues">Reporting issues</h1>
...
+        </td>
+    </tr>
+</table>
 
 <h2>Services</h2>
 
-<h3>*.minetest.net</h3>
-<ul>
-  <li><abbr title="Domain Name System">DNS</abbr>: celeron55</li>
-  <li>Approval of hosting things: celeron55</li>
```
Comment (sfan5) [inline development.html]: this no longer appears anywhere

Comment (rubenwardy) [inline development.html]: contact celeron55 covers it, I feel

**Comment by rubenwardy** [comment]
screenshot_2017-03-11_21-03-44

**Comment by rubenwardy** [comment]
Updated, it now looks like this:

screenshot_2017-03-11_21-07-11

**Pull request merged**